### PR TITLE
Precommit autoupdate

### PR DESCRIPTION
**Pull request by walkowif**
* [Workflow run](https://github.com/walkowif/tern/actions/runs/4253138517/jobs/7397610763)
* [Resulting commit](https://github.com/walkowif/tern/commit/83c5a05455729b5e3a65f73b5339f798426d6579)

### CODE REVIEW

**Comment by walkowif** [comment]
Wait, the opt-in feature is missing 🙂

**Review by arkadiuszbeer — APPROVED** [review]
LGTM

File: .github/workflows/version-bump.yaml [hunk 1]
```diff
@@ -58,12 +64,20 @@ jobs:
           echo "NEW_PKG_VERSION=${DESC_VERSION}" >> $GITHUB_ENV
         shell: bash
 
+      - name: Precommit autoupdate 🅿️
+        if: inputs.disable-precommit-autoupdate != 'true'
+        run: |
+          apt update && apt install -y python3-pip
+          pip3 install pre-commit
+          pre-commit --version
+          pre-commit autoupdate
```
[the remark this answers is not in the document]
Comment (walkowif) [reply .github/workflows/version-bump.yaml:73]: Yeah, definitely.

[the remark this answers is not in the document]
Comment (walkowif) [reply .github/workflows/version-bump.yaml]: Should be working now.

**Pull request merged**